themes/classic/templates/layouts/layout-right-column.tpl

```diff
@@ -35,3 +35,9 @@
     {hook h="displayContentWrapperBottom"}
   </div>
 {/block}
+
+{if $page.page_name == 'contact'}
+  {block name="right_column_contact"}
+    {widget name="ps_contactinfo" hook='displayLeftColumn'}
```

Comment:
And finally if this is the right layout then why is the `displayLeftColumn` hook called?

Reply:
It's not even a real hook, it's just used to select the "rich" template

/modules/ps_contactinfo/ps_contactinfo.php